src/System.Net.NetworkInformation/tests/FunctionalTests/NetworkFiles/dev

```diff
@@ -0,0 +1,4 @@
+Inter-|   Receive                                                |  Transmit
```

Comment:
My understanding was that we shouldn't add any test data in this repo. Instead you'll need to use the test-data repo and create a Nuget Package containing these files.